test/test_custom_ops.py

```diff
@@ -865,6 +865,14 @@ def foo(x: Tensor, y: Tuple[int, int]) -> Tensor:
 
             del foo
 
+        with self.assertRaisesRegex(ValueError, r"List\[int\]"):
+            # test that we propose a correct and supported type.
+            @custom_ops.custom_op(f"{TestCustomOp.test_ns}::foo")
+            def foo(x: Tensor, y: Tuple[int, int]) -> Tensor:
+                raise NotImplementedError
```

Comment:
Can you test that Tuple[int, float] for example doesn't lead to us erroneously suggesting List[int]?

Reply:
added. 

The type converter would convert Tuple[int, float] to List[int, float], but since it is not a supported type, we do not suggest it. 

Maybe we can suggest List[int, float, bool], but the code is not that smart yet. 

I tried `self.assertRaisesRegex(ValueError, r"^((?!example).)*$"):`, but somehow the regex is not working, so I didn't use assertRaisesRegex.

Reply:
Sounds good